components/browser/menu/src/main/java/mozilla/components/browser/menu/WebExtensionBrowserMenuBuilder.kt

```diff
@@ -22,8 +22,8 @@ import mozilla.components.browser.state.store.BrowserStore
  * @param webExtIconTintColorResource Optional ID of color resource to tint the icons of back and
  * add-ons manager menu items.
  * @param onAddonsManagerTapped Callback to be invoked when add-ons manager menu item is selected.
- * @param appendExtensionActionAtStart true if web extensions appear at the top (start) of the menu,
- * false if web extensions appear at the bottom of the menu. Default to false (bottom).
+ * @param appendExtensionSubMenuAtStart true if web extension sub menu appear at the top (start) of
```

Comment:
nit: I find this name and description confusing. :) Isn't it just switching the position of the `<- Add-ons` and `Add-onsManager` items? The extension actions (menu items) are always in-between those...

Reply:
This is meant for the add-on sub menu position in the main menu. Sub menu items positions are not affected (hence the name change from action -> sub menu).